apps/server/src/modules/task/mapper/task.mapper.spec.ts

```diff
@@ -30,10 +30,22 @@ const createExpectedResponse = (
 };
 
 describe('task.mapper', () => {
+	let module: TestingModule;
+
+	beforeAll(async () => {
```

Comment:
it actually is because we are using entity relations here. this requires meta information to be read (decorators). and that is only possible when we have an entity manager initialzed. that's why the module. without it we get runtime errors while assigning entity relations.